tasks/snapshot.yml

```diff
@@ -0,0 +1,35 @@
+version: '3'
+
+tasks:
+  directory:
+    desc: "Creates a snapshot of the current working directory with .git, files in .drainpipeignore, and .drainpipeignore excluded. The target directory should be provided as a cli argument e.g. task snapshot:directory -- /tmp/release"
```

Comment:
Another question about Task; how does it handle per-task help? I expected to be able to do something like `task <command> --help`. It looks like it can only do a single line description as a part of `task --list`, which means we can't just stuff a long description there.

Reply:
https://taskfile.dev/#/usage?id=display-summary-of-task